Followers of 241709263
API Rate Limit exceeded. Waiting...
Time needed: 904.18798399 s
Followers of 397489668
Time needed: 1.05265116692 s
Followers of 18378349
API Rate Limit exceeded. Waiting...
API Rate Limit exceeded. Waiting...
Time needed: 1810.60754919 s
Followers of 327577091
API Rate Limit exceeded. Waiting...
Time needed: 914.218698978 s
Followers of 7517052
API Rate Limit exceeded. Waiting...
API Rate Limit exceeded. Waiting...
API Rate Limit exceeded. Waiting...
API Rate Limit exceeded. Waiting...
Time needed: 3629.073385 s
Followers of 1305698738
Time needed: 0.994784116745 s
Followers of 22522178
API Rate Limit exceeded. Waiting...
API Rate Limit exceeded. Waiting...
Traceback (most recent call last):
  File "get_followers.py", line 64, in <module>
    for follower in limit_handled(tweepy.Cursor(twitterAPI.followers_ids, user_id=id_user).items(), counter):	
  File "get_followers.py", line 22, in limit_handled
    yield cursor.next()
  File "/home/brambilla/anaconda3/envs/mattia-env/lib/python2.7/site-packages/tweepy/cursor.py", line 197, in next
    self.current_page = self.page_iterator.next()
  File "/home/brambilla/anaconda3/envs/mattia-env/lib/python2.7/site-packages/tweepy/cursor.py", line 75, in next
    **self.kargs)
  File "/home/brambilla/anaconda3/envs/mattia-env/lib/python2.7/site-packages/tweepy/binder.py", line 250, in _call
    return method.execute()
  File "/home/brambilla/anaconda3/envs/mattia-env/lib/python2.7/site-packages/tweepy/binder.py", line 192, in execute
    six.reraise(TweepError, TweepError('Failed to send request: %s' % e), sys.exc_info()[2])
  File "/home/brambilla/anaconda3/envs/mattia-env/lib/python2.7/site-packages/tweepy/binder.py", line 190, in execute
    proxies=self.api.proxy)
  File "/home/brambilla/.local/lib/python2.7/site-packages/requests/sessions.py", line 512, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/brambilla/.local/lib/python2.7/site-packages/requests/sessions.py", line 622, in send
    r = adapter.send(request, **kwargs)
  File "/home/brambilla/.local/lib/python2.7/site-packages/requests/adapters.py", line 495, in send
    raise ConnectionError(err, request=request)
tweepy.error.TweepError: Failed to send request: ('Connection aborted.', error(104, 'Connection reset by peer'))
Followers of 300868924
API Rate Limit exceeded. Waiting...
Time needed: 910.426007986 s
Followers of 211106542
Time needed: 2.03102397919 s
Followers of 575160556
API Rate Limit exceeded. Waiting...
Time needed: 901.125607014 s
Followers of 407347022
API Rate Limit exceeded. Waiting...
Connection aborted by peer. Waiting...
API Rate Limit exceeded. Waiting...
Time needed: 2112.11302614 s
Followers of 36499730
Time needed: 4.11290001869 s
Followers of 57268394
API Rate Limit exceeded. Waiting...
Time needed: 906.579071045 s
Overall time needed: 80.6064655026 min
